### DISCUSSION TITLE
Welcome to td-ameritrade-api Discussions!

### DISCUSSION THREAD

**Discussion by areed1192** [areed1192/td-ameritrade-api#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pprivon** [comment]
Hello! I'm relatively new to Python but have enjoyed some smaller coding projects in the past. Looking to turn my manual trading processes into automated processes to free up some of the manual work I go through. I mostly trade options but some stocks. Looking forward to further development of this API and interacting with others in this project. 

Best Regards,
Peter
